runtime/caml/sys.h

```diff
@@ -42,7 +42,6 @@ CAMLextern value caml_sys_exit (value)
 CAMLnoreturn_end;
 
 extern double caml_sys_time_unboxed(value);
-CAMLextern value caml_sys_get_argv(value unit);
```

Comment:
Could we keep the old primitive for compatibility? (Or do we know that this is in the no-external-users part of the headers?)

Reply:
It's in the no-external-users part. (See `#ifdef CAML_INTERNALS` above)